package.json

```diff
@@ -268,6 +268,7 @@
   },
   "resolutions": {
     "elliptic": "^6.5.3",
-    "nwmatcher": "^1.4.4"
+    "nwmatcher": "^1.4.4",
+    "loader-utils": "^2.0.3"
```

Comment:
we've had runtime issues using overrides where we call APIs that have changed in the patched module.  Have you done runtime testing of the scenarios where this package is to confirm this new version is compatible?  Since it's a minor version bump it likely is I would guess but still should do runtime testing.